.github/workflows/publish-1.0-docs.yml

```diff
@@ -0,0 +1,68 @@
+name: Publish 1.0 docs
+
+permissions:
+  contents: read
+
+on:
+  workflow_dispatch:
+
+jobs:
+  publish-docs:
+    # runs on main repo only
+    if: github.repository == 'apache/incubator-pekko-persistence-jdbc'
+    name: Publish
+    runs-on: ubuntu-20.04
+    env:
+      JAVA_OPTS: -Xms2G -Xmx2G -Xss2M -XX:ReservedCodeCacheSize=256M -Dfile.encoding=UTF-8
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          # we don't know what commit the last tag was it's safer to get entire repo so previousStableVersion resolves
+          fetch-depth: 0
+
+      - name: Setup Java 8
+        uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 8
```

Comment:
Not 11?

Reply:
this is what we use in the nightly build - I don't want to use a different java version in this build

Reply:
iirc all of our projects aside from pekko core (due to technical reasons) are built with JDK 8